modules/nf-core/pilon/main.nf

```diff
@@ -40,4 +40,17 @@ process PILON {
     "${task.process}":
         pilon: \$(echo \$(pilon --version) | sed 's/^.*version //; s/ .*\$//' )
```

Comment:
END_VERSIONS is missing, probably not breaking anything but it looks weird

Reply:
I'd have expected it to fail at that point! Will fix it when I'm back on my computer